manifests/0000_30_cluster_csr_approver_02_clusterrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:operator:cluster-csr-approver-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
this doesn't look right.  I expected to see a clusterrole that has the correct permissions.

Reply:
+1, this has to be refined

Reply:
Fixed. Thanks for pointing this out!